tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -2368,11 +2368,11 @@ error: Unknown symbol \\"IDontExist\\"
    |                          ^^^^^^^^^^ Unknown symbol \\"IDontExist\\"
 
 
-error: Expected an interface, instead found type \\"ISomeClass\\"
+error: Unknown symbol \\"ISomeClass\\"
    --> ../../../examples/tests/invalid/interface.test.w:16:34
    |
 16 | interface ISomeInterface extends ISomeClass {
-   |                                  ^^^^^^^^^^ Expected an interface, instead found type \\"ISomeClass\\"
+   |                                  ^^^^^^^^^^ Unknown symbol \\"ISomeClass\\"
```

Comment:
This is a regression, but it will be fixed when we support proper class definition hoisting (intentionally left out of this PR for being out of scope).